vite.config.ts

```diff
@@ -20,6 +21,10 @@ for (let entry of entryFiles) {
 export default defineConfig({
   // esbuild in vite does not support decorators
   // https://github.com/evanw/esbuild/issues/104
+  test: {
+    // 👋 add the line below to add jsdom to vite
+    environment: 'jsdom',
```

Comment:
this library's tests run in node, firefox, and chrome -- so we may want to conditionally set this for node only